scripts/gha/it_workflow.py

```diff
@@ -62,6 +62,8 @@
 _COMMENT_TITLE_FAIL = "### ❌&nbsp; Integration test FAILED\n"
 _COMMENT_TITLE_SUCCEED = "### ✅&nbsp; Integration test succeeded!\n"
 
+_COMMENT_FLAKY_TRACKER = "Add flaky tests to **[go/fpl-cpp-flake-tracker](https://docs.google.com/spreadsheets/d/1KQdg062mY2vTGq2w8bQ15CXopuXKQ2jro3Eob8hBnLQ/edit?usp=sharing&resourcekey=0-cjzOgusCPXYDVKLc-zyLOA)**\n"
```

Comment:
The URL in the parenthesis should be `http1://go/fpl-cpp-flake-tracker`

Reply:
fixed. And I believe you mean `http://go/fpl-cpp-flake-tracker`

Reply:
Ah yes, that I did!